src/DynamoMLDataPipeline/BinaryAsset.cs

```diff
@@ -37,7 +37,7 @@ public BinaryAsset()
         public string Type { get; set; }
     }
 
-    class BinaryAssets
+    internal class BinaryAssets
     {
         [JsonProperty("binaries")]
```

Comment:
I think all these JsonProperties can be removed if you use serializer settings that set names to lower case.
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-8-0

less code to maintain is better.

Reply:
Will address this in the next version, as more changes are needed and additional testing would be needed.